libs/wingsdk/src/target-tf-aws/website.ts

```diff
@@ -35,6 +37,18 @@ export class Website extends cloud.Website {
 
     this.uploadFiles(this.path);
 
+    // create a cloudfront oac
+    const cloudfrontOac = new CloudfrontOriginAccessControl(
+      this,
+      "CloudfrontOac",
+      {
+        name: "cloudfront-oac",
```

Comment:
Can we make the bucket non-public?

Reply:
Isn't the S3 bucket already private by default?